packages/flame_bloc/CHANGELOG.md

```diff
@@ -1,3 +1,7 @@
+## 1.1.0
+
+ - Bump "flame_bloc" to `1.1.0`.
+
 # CHANGELOG
```

Comment:
You can remove this line and make the rest of the changelog conform to the new format, that's what we did on `flame_test`.